brave-grpc/pom.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>brave</artifactId>
+        <groupId>com.github.kristofa</groupId>
+        <version>3.5.1-SNAPSHOT</version>
+    </parent>
+    <artifactId>brave-grpc</artifactId>
+    <packaging>jar</packaging>
+    <name>brave-grpc</name>
+    <description>
+        gRPC integration that uses the brave api to submit client and server side span information.
+    </description>
+    <url>https://github.com/kristofa/brave</url>
+    <licenses>
+        <license>
+            <name>Apache 2</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <grpc.version>0.13.2</grpc.version>
+        <protobuf.version>3.0.0-beta-2</protobuf.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.github.kristofa</groupId>
+            <artifactId>brave-http</artifactId>
```

Comment:
this dep is only used to read constants (BraveHttpHeaders). I'd maybe make a comment like:

```
<!-- brave-http contains propagation header keys (BraveHttpHeaders) -->
```

Luckily, brave-http has no additional deps, so is harmless (beyond adding some number of KiB)